vendor.mod

```diff
@@ -210,3 +210,5 @@ require (
 	k8s.io/klog/v2 v2.90.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/Microsoft/hcsshim => github.com/anmaxvl/hcsshim v0.10.0-rc.1.0.20230914180317-064d46e61813
```

Comment:
This is mildly DNM; still waiting on a merge of https://github.com/microsoft/hcsshim/pull/1900

Reply:
this commit was dropped after https://github.com/moby/moby/pull/46522